README.md

```diff
@@ -53,6 +53,10 @@ Some can be changes in the key config dialog in the settings tab.
 | T                             | Chat                                                           |
 | /                             | Command                                                        |
 | Esc                           | Pause menu/abort/exit (pauses only singleplayer game)          |
+| C                             | Cycle through camera modes                                     |
+| Tab                           | Cycle through minimap modes                                    |
+| Shift + Tab                   | Change minimap orientation                                     |
+| F8                            | Toggle cinematic mode                                          |
```

Comment:
Why move?

Reply:
I agree, F8 should remain with the other F keys.